### PR TITLE
Core: update modules

### PR DESCRIPTION
**Pull request by Berserker66**
## What is this fixing or adding?
```
Traceback (most recent call last):
  File "__startup__.py", line 130, in run
  File "console.py", line 16, in run
  File "MultiServer.py", line 2294, in <module>
  File "asyncio\runners.py", line 190, in run
  File "asyncio\runners.py", line 118, in run
  File "asyncio\base_events.py", line 653, in run_until_complete
  File "MultiServer.py", line 2262, in main
NameError: name 'exit' is not defined
```

## How was this tested?
running MultiServer, Generate, Launcher and TextClient frozen on windows

### CODE REVIEW

File: MultiServer.py [hunk 1]
```diff
@@ -2254,12 +2254,15 @@ async def main(args: argparse.Namespace):
                 if not isinstance(e, ImportError):
                     logging.error(f"Failed to load tkinter ({e})")
                 logging.info("Pass a multidata filename on command line to run headless.")
-                exit(1)
+                # when cx_Freeze'd the built-in exit is not available, so we import sys.exit instead
```
Comment (black-sliver) [inline MultiServer.py:2257]: Wondering if it's worth reporting this as a bug? sys.exit is easy to forget when exit works in non-frozen.

Comment (Berserker66) [reply MultiServer.py:2257]: Done in https://github.com/marcelotduarte/cx_Freeze/issues/1937

**Comment by ThePhar** [comment]
It's looking like they don't want to "fix" it.

**Review by ThePhar — APPROVED** [review]
Considering what seems to be cx_freeze teams' stance, LGTM.

**Pull request merged**